cmd/kubelet/app/server.go

```diff
@@ -749,7 +750,7 @@ func run(ctx context.Context, s *options.KubeletServer, kubeDeps *kubelet.Depend
 		if err != nil {
 			return err
 		}
-		reservedSystemCPUs, err := getReservedCPUs(machineInfo, s.ReservedSystemCPUs)
+		reservedSystemCPUs, err := getReservedCPUs(klog.FromContext(ctx), machineInfo, s.ReservedSystemCPUs)
```

Comment:
Although passing logger is acceptable, I'd suggest to consider passing a context. Context generally is more useful and can be used not just to get a logger.

Reply:
This is a generic comment about the PR, not just for this particular case.